woeip/templates/includes/_header.html

```diff
@@ -0,0 +1,21 @@
+<!-- Navigation Bar -->
+<ul>
+    <li><h1><a href="{% url 'index' %}">West Oakland Air Quality</a></h1></li>
+    <li><a href="{% url 'files' %}">File Management</a></li>
```

Comment:
"File Management" seems like too broad a term -- I'd consider how this sounds to our target audience, the WOEIP users. "Upload a new session" might be more clear.